### PR TITLE
CEB-21: Added generic CustomEntityType with read-only code

### PR DESCRIPTION
**Pull request by rybus**

### CODE REVIEW

File: Form/Type/CustomEntityType.php [hunk 1]
```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace Pim\Bundle\CustomEntityBundle\Form\Type;
+
+use Pim\Bundle\EnrichBundle\Form\Subscriber\DisableFieldSubscriber;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\FormBuilderInterface;
+
+/**
```
Comment (fitn) [inline Form/Type/CustomEntityType.php]: Phpdoc

File: Form/Type/CustomEntityType.php [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        $builder
+            ->add('code')
+            ->addEventSubscriber(new DisableFieldSubscriber('code'));
+    }
+
+    public function getName()
+    {
+        return 'pim_enrich_custom_entity';
```
Comment (fitn) [inline Form/Type/CustomEntityType.php]: pim_custom_entity_reference_data ?

pim_enrich should not appear here I think. The form is in pim_custom_entity

File: Resources/config/form_types.yml [hunk 3]
```diff
@@ -0,0 +1,5 @@
+services:
+    pim_custom.form.type.custom_entity:
```
Comment (fitn) [inline Resources/config/form_types.yml]: pim_custom_entity.form.type.custom_entity or pim_custom_entity.form.type.reference_data ? As you prefer but we should rename classes accordingly

File: Form/Type/CustomEntityType.php (hunk 1 above)
Comment (rybus) [inline Form/Type/CustomEntityType.php]: done

**Comment by fitn** [comment]
GTM

**Pull request merged**